backend/satellite_tools/repo_plugins/yum_dnf_src.py

```diff
@@ -208,18 +208,16 @@ def setup_repo(self, repo, no_mirrors, ca_cert_file, client_cert_file, client_ke
         self.digest=hashlib.sha256(self.url.encode('utf8')).hexdigest()[:16]
         self.dnfbase.repos.add(repo)
         self.repoid = repo.id
-        try:
-            logger = logging.getLogger('dnf')
-            logger.setLevel(logging.ERROR)
-            self.yumbase.repos[self.repoid].load()
-            logger.setLevel(logging.WARN)
-        except RepoError:
```

Comment:
I have not checked the sources yet, but are you sure we can drop exception handling here while load() is still used?

Reply:
I believe the exception was to test whether a mirror link is working. Previously it threw an exception when it wasn't a mirror link (at least as far as I understand from the inline comment).
Now dnf automatically reverts back to base URL and only displays a warning. So there is no exception. My thought is that this removes the purpose of the original exception handling (check if a mirror link can be resolved).

Is there/can there be a different purpose based on the handled code?

Reply:
You are right, the try/catch block is required in case the URL or repo itself is not working. I will re-add it.